README.md

```diff
@@ -227,9 +227,9 @@ sql.notify('news', JSON.stringify({ no: 'this', is: 'news' }))
 
 This also means you cannot write dynamic queries or concat queries together by simple string manipulation. To enable dynamic queries in a safe way, the `sql` function doubles as a regular function which escapes any value properly. It also includes overloads for common cases of inserting, selecting, updating and querying.
 
-## Dynamic query helpers `sql() inside tagged template`
+## Dynamic query helpers - `sql()` inside tagged template
 
-Postgres.js has a safe, ergonomic way to aid you in writing queries. This makes it easier to write dynamic inserts, selects, updates and where queries.
+Postgres.js has a safe, ergonomic way to aid you in writing queries. This makes it easier to write dynamic inserts, selects, updates and `where` queries.
```

Comment:
Should probably remove the pluralization of insert, select, update too and put in quotes like `where` now that you're at it 😉